src/components/LoadFileMenu/index.tsx

```diff
@@ -117,18 +117,14 @@ const LoadFileMenu = ({
 
     return (
         <>
-            <Dropdown
-                menu={{ items, theme: "dark", className: styles.menu }}
+            <CustomDropdown
+                items={items}
+                titleText={"Load model"}
+                icon={DownArrow}
+                buttonType={ButtonClass.Primary}
                 placement="bottomRight"
                 disabled={isDisabled}
-            >
-                <NavButton
-                    titleText={"Load model"}
-                    icon={DownArrow}
-                    buttonType={ButtonClass.Primary}
-                    isDisabled={isDisabled}
-                />
-            </Dropdown>
```

Comment:
Really like that the NavButton gets collapsed into Dropdown in all the instances here :')